lib/tasks/knows_what_exercises_are_called_and_where_they_live_in_the_directory_structure.rb

```diff
@@ -0,0 +1,34 @@
+class Exercise
```

Comment:
Generally the right direction. 
Renaming a class to something more generic like that can be dangerous.
But maybe it's the right time to do it.
I like how you've made it instantiable.